File: testprojects/streaming/python/pipelines/StreamingTestPipeline_1/code/streamingtestpipeline_1/graph/Limit_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingtestpipeline_1.config.ConfigStore import *
from streamingtestpipeline_1.udfs.UDFs import *

def Limit_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.limit(10)

File: testprojects/scala_main/pipelines/SCALA_VECTOR_TYPE/code/src/main/scala/io/prophecy/pipelines/scala_vector_type/graph/Subgraph_1/Limit_2.scala
package io.prophecy.pipelines.scala_vector_type.graph.Subgraph_1

import io.prophecy.libs._
import io.prophecy.pipelines.scala_vector_type.udfs.UDFs._
import io.prophecy.pipelines.scala_vector_type.graph.Subgraph_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Limit_2 {
  def apply(context: Context, in: DataFrame): DataFrame = in.limit(10)
}